src/command/uninstall.rs

```diff
@@ -15,10 +14,8 @@ impl Command for Uninstall {
     fn run(self, session: &mut Session) -> Fallible<ExitCode> {
         session.add_event_start(ActivityKind::Uninstall);
 
-        let version = VersionSpec::default();
-        let tool = tool::Spec::from_str_and_version(&self.tool, version);
-
-        tool.uninstall()?;
```

Comment:
This `uninstall` is a function of `Spec`. ↓

https://github.com/volta-cli/volta/blob/17763872574f6c67841282551bc38609a11a9f00/crates/volta-core/src/tool/mod.rs#L120-L124

What I want to do now is to perform an uninstall with a specified version like `volta uninstall node@x.x.x`, so `Spec::uninstall` is not appropriate. Therefore, I considered defining `Tool::uninstall`. (Since `Tool` is the return type of `Spec::resolve`, I believe that "Resolved" mentioned in this comment refers to `Tool`).

But I'm not sure that I can replace `Spec::uninstall` with the `Tool::uninstall` completely.